src/ynEIGEN/EigenStrategyManager.sol

```diff
@@ -288,55 +418,33 @@ contract EigenStrategyManager is
      * @param assets An array of ERC20 tokens for which balances are to be retrieved.
      * @return stakedBalances An array of total balances for each asset, indexed in the same order as the `assets` array.
      */
-    function getStakedAssetsBalances(IERC20[] calldata assets) public view returns (uint256[] memory stakedBalances) {
+    function getStakedAssetsBalances(IERC20[] calldata assets) public view returns (uint256[] memory) {
 
-        stakedBalances = new uint256[](assets.length);
+        uint256[] memory stakedBalances = DynamicArrayLib.malloc(assets.length);
```

Comment:
Second thoughts about this:

why use this dynamicArraylib?

this is an in memory array - it's cheap to write to anyway.

It's also short in length.

I don't see any reason why we would use this library for this situation

Reply:
https://github.com/yieldnest/yieldnest-protocol/pull/164/commits/d44da58c13d8bf38e040c44565bb12b833c6e797